bundle-workflow/README.md

```diff
@@ -216,3 +217,17 @@ The following options are available.
 |--------------------|-------------------------------------------------------------------------|
 | --keep             | Do not delete the temporary working directory on both success or error. |
 | -v, --verbose      | Show more verbose output.                                               |
+
+### Onboarding Requirements
+
+With many components included in the distribution each component has the responsibility to keep the release process streamlined and react to integration issues.
+
+- Components repositories must have a tag matching all manifest versions so issue can be appropriately tagged.
```

Comment:
I would prefer not to use the word "must", because everything is a must here. Just say "Component reponsitories create tags that match ..., for example ..."

Reply:
labelled! Thanks